src/main/resources/reference.conf

```diff
@@ -7,11 +7,21 @@ gargoyle {
 
     ranger {
         # Settings for reaching Ranger.
+
+        # make sure the service_type is equal to what is specified in
+        # ranger-s3-security.xml
+        service_type = "s3"
+        app_id = "testservice"
     }
 
     storage.s3 {
         # Settings for reaching backing storage.
         host = "127.0.0.1"
```

Comment:
for windows a docker runs in VirtualBox and I have a different IP so it would be great to have an env varaible here like e.g GARGOYLE_S3_HOST = 
because now I have to change manualy the config:

    storage.s3 {
        # Settings for reaching backing storage.
        host = "192.168.99.100"
        port = 8010
    }

    sts {
        host = "192.168.99.100"
        port = 12345
    }

of course the defaults are here and the envs in application.conf

Reply:
done